graphql-server/src/middlewares/logging.ts

```diff
@@ -0,0 +1,43 @@
+import { Express } from "express";
+import path from "path";
+const morgan = require("morgan");
```

Comment:
import?

Reply:
For Angelica's context: 

Replacing `const morgan = require("morgan");` with `import morgan from "morgan";` results in the Typescript error "TS7016: Could not find a declaration file for module '<module-name>'. '<module's index.js file>' implicitly has an 'any' type. Try `npm install @types/<module-name>` if it exists or add a new declaration (.d.ts) file containing `declare module '<module-name>';`.

Yesterday, Aaron and I followed the error message's tip to "add a new declaration (.d.ts) file containing `declare module '<module-name>';". This made the error message go away.

Reply:
@alisman When I tried testing this today and run the local dev server with `yarn dev:backend`, the same error message reappeared. 

As a fix, I followed a StackOverflow post and tried including the path to the declaration file inside `tsconfig.json` but saw the same result.

I still like the idea of using `import` instead of `require` everywhere for its clean look, but I don't think there would be any added practical benefits because we're not defining the package types. 

I'd like to revisit this task if we do need to define the types in the future. For now, I'm dropping this task but happy to keep investigating if you think otherwise.